images/scripts/commons/update.sh

```diff
@@ -29,3 +29,4 @@ apt-get -y update
 apt-get -y upgrade -o Dpkg::Options::="--force-confnew"
 
 reboot
+sleep 120
```

Comment:
A loop is better because this way we don't need to know how long reboot waits for graceful shutdowns.
```suggestion
while true; do
  sleep 1
done
```